google/api_core/bidi.py

```diff
@@ -717,6 +719,7 @@ def stop(self):
                     _LOGGER.warning("Background thread did not exit.")
 
             self._thread = None
+            self._on_response = None
```

Comment:
This assumes you can't restart an instance after it has been stopped. The test cases and docstrings point to that being the case, but it's not completely clear